src/Persistence/Templates/ControlFactory.cs

```diff
@@ -17,11 +17,15 @@ public ControlFactory(IControlTemplateStore controlTemplateStore)
         _controlTemplateStore = controlTemplateStore ?? throw new ArgumentNullException(nameof(controlTemplateStore));
     }
 
-    public Control Create(string name, string template, string? variant = null, ControlPropertiesCollection? properties = null, IList<Control>? children = null)
+    public Control Create(string name, string template, string? componentInstance = null, string? variant = null,
```

Comment:
`componentInstance` seems to be the `componentDefinitionName`, is that right?

Reply:
Good point. I'll rename to make it clearer